scala/src/main/scala/smile/clustering/distance/Binary/PatternDifference.scala

```diff
@@ -0,0 +1,16 @@
+package smile.math.distance
+
+import scala.math.pow
+
+/**
+ * @author Beck Gaël
+ *
+ **/
+class PatternDifference extends BinaryDistance {
```

Comment:
Please add comment what this is, how it is computed, and what use cases it is good for.

Reply:
I don't know good use case, i just implemented it in order to test them. Did i need to remove them ?